lecture2/code.py

```diff
@@ -0,0 +1,133 @@
+#######################################################
+"""Personal Tests Below This rectanle! """
+#######################################################
+"""
+def concatinate_strings(string1, string2):
+    string1 = "Hello"
+    string2 = "World"
+    string_concat = ""
+    if string_concat != "HelloWorld":
+        return string1 + string2
+    else:
+        return string1, string2 and string_concat            
+
+2nd attempt at using the not in check in Python
+
+def concatinate_strings(string1, string2):
+    string1 = "Hello"
+    string2 = "World"
+    string_concat = ""
+    if "HelloWorld" not in string_concat:
+        string_concat = string1 + string2
+        return string_concat
+    if "Hello" not in string2:
+        return string1
+    if "World" not in string1:
+        return string2          
+    else:
+        return ""
+
+"""
```

Comment:
I see what you were thinking here and hope you can avoid thinking so complicated. Checks in this point do not benefit at all.